contracts/ConsumerHost.sol

```diff
@@ -51,6 +51,11 @@ contract ConsumerHost is Initializable, OwnableUpgradeable, IConsumer, ERC165 {
     /// @notice StateChannels' belongs to consumer
     mapping(uint256 => address) public channels;
 
+    /// @notice controller account belongs to consumer
+    mapping(address => address) private controllerConsumer;
+    mapping(address => address) private consumerController;
```

Comment:
why need two mapping ? 
If we just use `consumerControllers`, when checkSign, check the `sConsumer` is consumer self (`channels[channelId]`), or `consumerControllers[consumer]`